spec/acceptance/candlepin_spec.rb

```diff
@@ -10,11 +10,9 @@
         ensure => present,
       }
 
-      ['/usr/share/tomcat/conf', '/etc/candlepin/certs/amqp'].each |$dir| {
```

Comment:
Something is failing in the acceptance tests:
```
  Error: Could not set 'present' on ensure: No such file or directory @ rb_sysopen - /etc/candlepin/certs/candlepin-ca.key (file: /etc/puppetlabs/code/modules/certs/manifests/keypair.pp, line: 18)
```

Looks like the certs directory is still used for the regular files (`candlepin_certs_dir`)